backend/app/GraphQL/Mutations/AddMember.php

```diff
@@ -15,10 +15,10 @@ final class AddMember
      */
     public function __invoke($_, array $args)
     {
-        $authId = Auth::id();
+        $authUser = Auth::user();
         $team = Team::find($args['team_id']);
 
-        if ($authId != $team->teamLeader->id) {
+        if ($authUser->id != $team->teamLeader->id && $authUser->role_id != 1) {
```

Comment:
I think there is a conflict for this condition since a user can only be assigned as a Team Leader if they have the role_id: 2 (team leader)

Might be better to create the permissions if they don't exist yet and use the `hasPermission` directive.

Reply:
I think applying roles and permissions is for next sprint so i did not added it yet